tox.ini

```diff
@@ -33,6 +33,24 @@ commands =
     coverage run tests/manage.py test testapp.tests {posargs: -v 2}
     coverage report -m
 
+[testenv:interactive]
+basepython = python3.9
+
+;deps =
+;    wagtail>=2.16,<2.17
```

Comment:
Why is this commented out? If the intention is to reinstate it for development purposes, could we add a comment explaining that? If not, it should probably be removed.

Reply:
that is indeed its only purpose.